docker.go

```diff
@@ -79,16 +94,36 @@ func (container *Container) Volume(hostPath, containerPath, mode string) *Contai
 func NewDockerContainer() *Container {
 	container := new(Container)
 	container.Type = "DOCKER"
-	container.Docker = &Docker{
-		Image:        "",
-		Network:      "BRIDGE",
-		PortMappings: make([]*PortMapping, 0),
-		Parameters:   make([]*Parameters, 0),
-	}
-	container.Volumes = make([]*Volume, 0)
+	container.EmptyVolumes()
+	container.Docker = &Docker{}
+	container.Docker.Container("").
+		Bridged().
+		SetPrivileged(false).
+		SetForcePullImage(false).
+		EmptyParameters().
+		EmptyPortMappings()
+
 	return container
 }
 
+// SetForcePullImage sets whether the docker image should always be force pulled before
+// starting an instance
+//		forcePull:			forceImagePull true / false (false by default)
```

Comment:
If the `(false by default)` refers to Marathon's default, then I would leave it out as it could change over time. Clients should read up on the official REST documentation in order to learn about the default behavior.